docs/reference/monitoring/configuring-filebeat.asciidoc

```diff
@@ -0,0 +1,190 @@
+[role="xpack"]
+[testenv="basic"]
+[[configuring-filebeat]]
+=== Collecting {es} log data with {filebeat}
+
+[subs="attributes"]
+++++
+<titleabbrev>Collecting log data with {filebeat}</titleabbrev>
+++++
+
+You can use {filebeat} to monitor the {es} log files, collect log events, and
+ship them to the monitoring cluster. In 7.2 and later, your recent logs are
```

Comment:
Is this availability date information is really necessary? This page in this form will only exist in 7.2 and later documentation, right? So, it might be redundant to add this information.

Reply:
I think we've included that in monitoring instructions in the past because your monitoring cluster can be at a different version.release than your production cluster. But I think you're right that it's not really necessary.